pytest_fixtures/content_hosts.py

```diff
@@ -0,0 +1,95 @@
+"""
+This module is to define pytest functions for content hosts
+
+The functions in this module are read in the pytest_plugins/fixture_markers.py module
+All functions in this module will be treated as fixtures that apply the contenthost mark
+"""
+import pytest
+from broker.broker import VMBroker
+
+from robottelo.config import settings
+from robottelo.hosts import ContentHost
+
+
+@pytest.fixture
```

Comment:
I believe this decorator needs removed, as it was the cause of a scope mismatch exception.